DunglasActionBundle.php

```diff
@@ -23,6 +23,11 @@
      */
     public function build(ContainerBuilder $container)
     {
-        $container->addCompilerPass(new RegisterCompilerPass());
+        $passConfig = $container->getCompilerPassConfig();
```

Comment:
The problem here is that the framework bundle read the tag on ``TYPE_BEFORE_OPTIMIZATION``. 
So that's the only way for now to be sure that our pass is executed before.